app/assets/javascripts/pretty-text/addon/engines/discourse-markdown-it.js

```diff
@@ -578,14 +579,16 @@ export function setup(opts, siteSettings, state) {
 
 export function cook(raw, opts) {
   // we still have to hoist html_raw nodes so they bypass the allowlister
-  // this is the case for oneboxes
-  let hoisted = {};
-  opts.discourse.hoisted = hoisted;
+  // this is the case for oneboxes and also certain plugins that require
+  // raw HTML rendering within markdown bbcode rules
+  opts.discourse.hoisted = opts.discourse.hoisted || {};
```

Comment:
Not sure if we use nullish assignment operator, but it's supported in all mainline browsers:

```suggestion
  opts.discourse.hoisted ??= {};
```

Reply:
Yes we do, took a look and it's used a few times in the codebase, thanks for the tip :+1: